packages/autorest.typescript/src/generators/static/packageFileGenerator.ts

```diff
@@ -118,7 +118,8 @@ function regularAutorestPackage(
       "rollup-plugin-sourcemaps": "^0.6.3",
       typescript: "~4.8.0",
       "uglify-js": "^3.4.9",
-      rimraf: "^3.0.0"
+      rimraf: "^3.0.0",
+      dotenv: "^8.2.0"
     },
     // TODO: Calculate the SDK path for the package
     homepage: `https://github.com/Azure/azure-sdk-for-js/tree/main/${azureOutputDirectory}`,
```

Comment:
Don't see any logic to check if the azureOutputDirectory is undefined or not

Reply:
It seems that this is a HLC bug